tasks/main.yml

```diff
@@ -36,12 +36,5 @@
     group: root
     mode: 0744
 
-- name: Enable traefik service at boot
-  service:
-    name: traefik
-    enabled: yes
-
-- name: Ensure traefic service is running
-  service:
-    name: traefik
-    state: started
+- name: Ensure traefik service is enabled and running
+  service: name=traefik state=started enabled=yes
```

Comment:
@sjourdan Could you remove task inline config (name=...) and put each option on a single line to be ?

To be more homogenous with the rest of the role.